### PR TITLE
HLT menu development for `13_3_X` (1/N) [`14_0_X`]

### PR DESCRIPTION
**Pull request by mmusich**
#### PR description:

Development of HLT menus for CMSSW_13_3_X:
   * this is an intermediate update towards the 2024 proton-proton run data-taking
   
This is the PR for the master branch, which will then be backported down to 13_3_X (the latter is the release cycle currently used for Run-3 HLT-menu development).

This update includes the integration of the following JIRA tickets:
- [CMSHLT-2975](https://its.cern.ch/jira/browse/CMSHLT-2975) [EXO,TSG]: Bug in configuration of `*...NoFiltersNoVertex...*` HLT/MC paths
- [CMSHLT-2982](https://its.cern.ch/jira/browse/CMSHLT-2982) [EXO,TSG]: Misconfiguration of `hltL3fL1DoubleMuf0L2NVf15f7L3SingleMuNVf18DisplacedHybDxy0p1cm`
- [CMSHLT-2979](https://its.cern.ch/jira/browse/CMSHLT-2979) [EXO,HIG,TSG]: Misconfiguration of `hltDiPFJet45MJJ1050` (VBF-parking triggers)
- [CMSHLT-2985](https://its.cern.ch/jira/browse/CMSHLT-2985) [EXO,TSG]: Set `hltStoppedHSCPIterativeCone4CaloJets.srcPVs `to empty `InputTag`
- [CMSHLT-2986](https://its.cern.ch/jira/browse/CMSHLT-2986) [MUO,TSG]: Remove refs to offline products in `MuonIdProducer` instances at HLT

In addition it integrates in the combined table "Special" FOG trigger paths for non-physics production menus and updates table and testing scripts accordigly.
This addresses the following JIRA tickets:
- [CMSHLT-2860](https://its.cern.ch/jira/browse/CMSHLT-2860) [TSG]: The FOG "Ubermenu" (merge of FOG's "special" menus)
- [CMSHLT-2972](https://its.cern.ch/jira/browse/CMSHLT-2972) [TSG]: Circulating, Cosmics, CRUZET, MWGR -- port to 13_3_X

#### PR validation:

TSG integration tests.

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Not a backport, but it will be backported to CMSSW_13_3_X.

### CODE REVIEW

**Comment by mmusich** [comment]
enable gpu

**Comment by mmusich** [comment]
@cmsbuild, please test

File: HLTrigger/Configuration/test/cmsDriver.csh [hunk 1]
```diff
@@ -49,6 +50,7 @@ set InputLHCRawPIon2 = $InputLHCRawGRun2
 set InputLHCRawPRef2 = $InputLHCRawGRun2
 set InputLHCRawPIon3 = $InputLHCRawGRun3
 set InputLHCRawPRef3 = $InputLHCRawGRun3
+set InputLHCRawSpec3 = /store/data/Run2023D/Commissioning/RAW/v1/000/370/772/00000/f4fb5fb9-c8b8-48cd-9281-c23902260c92.root
```
Comment (Martin-Grunewald) [inline HLTrigger/Configuration/test/cmsDriver.csh:53]: Why not $InputLHCRawGRun3 ?

Comment (mmusich) [reply HLTrigger/Configuration/test/cmsDriver.csh:53]: This was one of the files suggested by FOG as used in their integration tests. No other particular reason.

Comment (Martin-Grunewald) [reply HLTrigger/Configuration/test/cmsDriver.csh:53]: I think the GRun3 file would be better as it contains more physics objects firing more of the triggers.

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by Martin-Grunewald** [comment]
+1

**Comment by Martin-Grunewald** [comment]
urgent
for pre3

**Comment by Martin-Grunewald** [comment]
@cms-sw/orp-l2 Could you merge this PR for pre3?

**Comment by antoniovilela** [comment]
+1

**Pull request merged**